.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    name: Build app
```

Comment:
El name del Job podría ser Build, lint and test pues son las tres acciones que se realizan